tests/fix_pq.py

```diff
@@ -53,7 +53,11 @@ def libpq():
         # Not available when testing the binary package
         libname = find_libpq_full_path()
         assert libname, "libpq libname not found"
-        return ctypes.pydll.LoadLibrary(libname)
+        if sys.implementation.name == "pypy" and not hasattr(ctypes, "pydll"):
+            # https://foss.heptapod.net/pypy/pypy/-/issues/3496
+            return ctypes.cdll.LoadLibrary(libname)
+        else:
+            return ctypes.pydll.LoadLibrary(libname)
```

Comment:
Python's documentation states:

> Thus, this is only useful to call Python C api functions directly.

See https://docs.python.org/3/library/ctypes.html#ctypes.PyDLL

Given this is calling into libpq, should this just be using CDLL everywhere?

Reply:
Looking at 69fbbec4337494550d76de9837717534e6f70e9b and to the fact that [in the main adapter we use `ctypes.cdll`](https://github.com/psycopg/psycopg/blob/607425c28ba6644163dc9b876f31ac2f480ec81f/psycopg/psycopg/pq/_pq_ctypes.py#L21), I think we can use `cdll` everywhere.